jupyterlab_git/git.py

```diff
@@ -870,14 +872,33 @@ async def init(self, current_path):
         Execute git init command & return the result.
         """
         cmd = ["git", "init"]
+        cwd = os.path.join(self.root_dir, current_path)
         code, _, error = await execute(
-            cmd, cwd=os.path.join(self.root_dir, current_path)
+            cmd, cwd=cwd
         )
+        if code == 0:
+            code, _, error = await self._maybe_run_actions('post_init', cwd)
```

Comment:
Opened question: Should stdout logged to help debugging?

Reply:
Do we have any examples of logging? For now, I sent all data (stdout/stderr/code/command) back to the front end where we can later show whatever we'd like to the user.